msi/release.sh

```diff
@@ -1,17 +1,56 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+me=$(basename $0)
+function showUsage() {
```

Comment:
Could you skip the `function` keyword, rename the function to `usage` and add a space between `usage` and `()`?